src/app/modules/shared/components/header/header.component.html

```diff
@@ -31,6 +31,9 @@
             <a class="nav-link" [routerLink]="['./general-instructions']">General Instructions</a>
           </li>
         </ul>
+        <button routerLink="/download-questions" class="btn btn-outline-success" type="button">
+          Candidate Registration
+        </button>
```

Comment:
Don't need to add this Candidate-Registration again, it's already in navbar with too validation